code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -170,12 +170,11 @@
 					M << "<span class='danger'>You feel a blast of paralyzingly cold air wrap around you and flow past, but you are unaffected!</span>"
 					continue
 			M << "<span class='userdanger'>A wave of shockingly cold air engulfs you!</span>"
-			M.Stun(2)
-			M.apply_damage(10, BURN)
```

Comment:
whoops, missed that one.

well it's a nerf instead of a buff and that was the main concern right